load.zsh

```diff
@@ -1,2 +1,18 @@
 #!/usr/bin/env zsh
+local dir=$(dirname $0)
+local olddir=$PWD
+
+cd $dir # logging directory
 source derby.zsh
+derby&
+
+export valet2k_ng=$PWD/nailgun/ng
+
+cd assistant
+# this test doesn't work - not sure why yet
+pgrep -f com.github.valet2k || mvn exec:java&
```

Comment:
this is the part where we need to have mvn on the path already, or download and specify the path